wercker.yml

```diff
@@ -1,4 +1,4 @@
-box: chaindev/ci:20170306
+box: chaindev/ci:20170405
```

Comment:
although, looks like I'll have to build a new docker image and update this again, since there's a merge conflict.